eval_test.go

```diff
@@ -204,3 +205,59 @@ func TestEvaluateNil(t *testing.T) {
 		})
 	}
 }
+
+func TestEvaluateNull(t *testing.T) {
```

Comment:
It's okay to add a test here. But I want to list the common use cases in `ExampleEvaluate` for documentation purpose.

![image](https://user-images.githubusercontent.com/1063615/219002399-bc98c5f8-3204-4629-88aa-65e4b7e5aaa0.png)